packages/teams-js/test/private/externalAPpCardActionsForDA.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { errorLibraryNotInitialized } from '../../src/internal/constants';
```

Comment:
Should this file name be externalA**p**pCardActionsForDA.spec.ts?

Reply:
Strange, I thought I fixed it. The vscode shows both names for some reason
![image](https://github.com/user-attachments/assets/4ae676bf-d913-4f0c-9218-9c047feddd0e)
Let me see if deleting and recreating the file would work